tools/min-base-version

```diff
@@ -28,14 +28,14 @@ from glob import glob
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
 max_version = '0'
-for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json')):
+for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json.in')):
```

Comment:
tought about this in issue #5997, and I think I have a reasonably good solution for this (at least much better than what we have now). So I think *this* particular hunk could stay, but I'm happy to revert the bumping of the dependency in `pkg/dashboard/manifest.json.in` (i. e. remove the topmost commit here) in this PR, as it will not actually influence the dependency of `-dashboard` (that doesn't use the manifest dependencies, it's just hardcoded at "same version as -ws") and unecessarily bump the dependency of all "plugin" packages.

This should alleviate the dependency concerns but still keep the unbreaking of min-base-version.